### PR TITLE
Tables

### LINKED ISSUE #10: Generic table component

### PR DESCRIPTION
**Pull request by mattgrill**
Fixes #10

### CODE REVIEW

**Comment by dawehner** [comment]
I addressed a good bunch of my points. I have one question left ...

File: src/components/table/sortable/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      rows,
+      headers: headers.map(header => ({
+        sortDirection: this.state.headers.filter(
+          stateHeader => stateHeader.key === header.header,
+        )[0].sortDirection,
+        ...header,
+      })),
+    });
+  }
+  sort = (key, direction) => {
```
Comment (dawehner) [inline src/components/table/sortable/index.js]: I would kinda expect to be able to swap out this behaviour with a HTTP based sorting. Any thoughts?

Comment (mattgrill) [reply src/components/table/sortable/index.js]: Yep. I added support for bypassing this functionality if you provide your own callback.

**Comment by dawehner** [comment]
Seems raesonable!

**Pull request merged**